opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/core/ExpressionScript.java

```diff
@@ -107,7 +113,9 @@ public Object visitParse(ParseExpression node, Set<ReferenceExpression> context)
 
   private OpenSearchExprValueFactory buildValueFactory(Set<ReferenceExpression> fields) {
     Map<String, OpenSearchDataType> typeEnv = fields.stream().collect(toMap(
-        ReferenceExpression::getAttr, e -> OpenSearchDataType.of(e.type())));
+        ReferenceExpression::getAttr, e -> {
```

Comment:
Also can be reverted (here and imports).

Reply:
Reverted in https://github.com/opensearch-project/sql/pull/1664/commits/1591a85db1e5627db9601b44b5e3d66079a9138f